assets/weapons/swb/colt/sounds/colt.fire.sound

```diff
@@ -1,19 +1,17 @@
 {
-  "UI": false,
-  "Volume": "1.00,1.00,0",
-  "Pitch": "1.00,1.00,0",
+  "UI": true,
+  "Volume": "0.5",
```

Comment:
The problem with sounds atm is that client sounds are played in UI mode and world sounds are not so that the client doesn't hear this strange sound origin. The result is that world gun sounds are too soft (for which there is an issue), you making them softer will result into even softer gun sounds.